content/guests.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Elm Radio Guest Instructions
+type: page
+---
+
+Here are a few notes about how Jeroen and Dillon prepare for and record Elm Radio episodes.
+
+## Preparation
+
+- We start with some high-level bullet points to outline the episode in a Notion document. That helps us structure the conversation. Other than that, it's fairly conversational and free-flowing.
+- The structure isn't set in stone and we often jump around, but it gives us a solid foundation.
+- The shows are edited, but only for ums, uhs, silences, and re-taking a sentence. Otherwise, it's just our direct conversation.
+
+## Outline Structure
+
+We usually start our outlines by filling in the following high-level areas for an episode.
+
+### What?
+
+Before we get into any details, we want the listener to know what the heck we're talking about. Is it an NPM package, an editor plugin, an Elm package, all 3? Introduce some basic definitions (if it's an AST package, what is an AST?).
+
+### Why?
+
+Before we talk about best practices and basics, let's give some motivation for the problem.
+
+### Best practices
+
+The medium of podcasts is great for helping people frame how they think about a tool or technique. It's also great for sharing useful tips and tricks. So let's use the podcast medium at what it's best for!
+
+### How to get started
+
+We like to leave listeners with our favorite
+
+- Resources
+- Ways to learn
+- Advice when you're getting started
+
+This helps them turn what they listened to into action and gives some inspiration to try something new.
+
+## Recording equipment
+
+### Essentials
+
+- Some sort of headphones (I prefer earbuds so I can hear my own voice). That way we don't get audio feedback from your speakers
+- We'll send you a link at the time of the recording to Zencastr. You'll just have to allow it to access your mic, and then we'll be able to download the audio from your mic as an independent track. No other work needed on your end
+- We have Zoom open during the call for video
```

Comment:
Mention Zoom before Zencastr, and that this is where we'll meet and start organizing the rest.